dev/run-dependencies.sh

```diff
@@ -14,6 +14,9 @@ export MONTAGU_ORDERLY_PATH=$(realpath $here/../src/app/git)
 export ORDERLY_SERVER_USER_ID=$UID
 $here/../scripts/run-dependencies.sh
 
+#$here/cli.sh add-user "test.user@example.com"
```

Comment:
This all started because I wanted to give the test user report reading permissions when running dev dependencies. Once this is merged can uncomment these lines.